app/schemas/account.py

```diff
@@ -24,6 +32,21 @@ class AccountCreateResponse(AccountBase):
     pass
 
 
+class UserCreate(UserBase):
+    pass
+
+
+class User(UserBase):  # UserBase를 상속받음
```

Comment:
Please separate user related schemas to a dedicated file?

Reply:
I moved it to schemas/user.py